### PR TITLE
State user home

### PR DESCRIPTION
**Pull request by benjaminpaige**
Added open sans fonts
Added state home page
Updated Table 
Updated Kebab Comp.

image

re ticket: https://qmacbis.atlassian.net/browse/OY2-8106

test: https://dqhfb84t04uec.cloudfront.net/

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/components/Footer/index.tsx [hunk 1]
```diff
@@ -9,20 +9,28 @@ export function Footer(): JSX.Element {
...
             For technical questions regarding use of this application, please
             reach out to{" "}
-            <a href={`mailto:${Libs.helpDeskContact.email}`}>
+            <CUI.Link
+              color="blue.500"
+              href={`mailto:${Libs.helpDeskContact.email}`}
+            >
               {Libs.helpDeskContact.email}
-            </a>
+            </CUI.Link>
```
Comment (jdinh8124) [inline services/ui-src/src/components/Footer/index.tsx:24]: Thanks for making the footer look more accurate!

**Review by jdinh8124 — DISMISSED** [review]
Demo looks awesome, really excited to see everything come together

File: services/ui-src/src/views/StateHome/__tests__/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <RouterWrappedComp>
+      <StateHome />
+    </RouterWrappedComp>
+  );
+});
+
+describe("Test StateHome", () => {
+  test("Check that the Heading renders", async () => {
+    expect(
+      await screen.findByText(/Core Set Measures Reporting/i)
```
Comment (13bfrancis) [inline services/ui-src/src/views/StateHome/__tests__/index.test.tsx:27]: Could await findby be replaced with this?
```suggestion
      screen.getByText(/Core Set Measures Reporting/i)
```
This applies to the below ones as well. If not, that is fine

**Review by 13bfrancis — APPROVED** [review]
This looks really good to me. I just added one comment

File: services/ui-src/src/views/StateHome/__tests__/index.test.tsx (hunk 2 above)
Comment (benjaminpaige) [reply services/ui-src/src/views/StateHome/__tests__/index.test.tsx:27]: yet it can, and so it shall 👍  way better

**Review by jdinh8124 — APPROVED** [review]

**Review by jdinh8124 — APPROVED** [review]

**Pull request merged**